tools/packaging/common/istio-iptables.sh

```diff
@@ -62,21 +62,23 @@ function dump {
 #
 #  Function check if argument is a valid ipv4 or ipv6 address
 #
+ipv4regexp="^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$"
+ipv6regexp="^[0-9a-fA-F]{1,4}:([0-9a-fA-F]{0,4}:){0,6}[0-9a-fA-F]{1,4}$"
```

Comment:
this won't match "::1" so I think you just change it to 
```
ipv6regexp="^([0-9a-fA-F]{0,4}:){0,7}[0-9a-fA-F]{1,4}$"
```

Reply:
Fixed, thank you for catching it

Reply:
Actually, per our slack discussion the way you have regex makes sense for this file since localhost wouldn't be used in the included/excluded ranges and any addresses would lead with a uni/any/multi-cast or linklocal routing prefix.

Reply:
removed in the latest revision..